tox.ini

```diff
@@ -62,3 +62,5 @@ setenv =
 passenv =
 commands =
     django-admin makemigrations --check -v3 --dry-run --noinput pinax_stripe
+deps =
```

Comment:
Once https://github.com/pinax/pinax-stripe/pull/631 is merged, this can be removed.